server/src/main/java/org/eclipse/lsp/cobol/core/visitor/VisitorHelper.java

```diff
@@ -185,7 +187,7 @@ private String retrieveIntervalText(@Nonnull ParserRuleContext ctx) {
    * @param positions map of exact positions
    * @return locality which has a range from the start to the end of the rule
    */
-  Optional<Locality> retrieveRangeLocality(
+   Optional<Locality> retrieveRangeLocality(
```

Comment:
Seems that you don't need this extra space here.

Reply:
Fixed